hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestHddsDispatcher.java

```diff
@@ -108,13 +108,12 @@ public class TestHddsDispatcher {
 
   @ContainerLayoutTestInfo.ContainerTest
   public void testContainerCloseActionWhenFull(
-      ContainerLayoutVersion layout) throws IOException {
+      ContainerLayoutVersion layout, @TempDir File testDir) throws IOException {
```

Comment:
This works fine, but we could simplify by adding it as member variable instead of parameter in each method.